channel/src/main/java/com/synopsys/integration/alert/channel/jira/server/distribution/JiraServerProcessorFactory.java

```diff
@@ -56,6 +56,7 @@ public JiraServerProcessorFactory(
 
     @Override
     public IssueTrackerProcessor<String> createProcessor(JiraServerJobDetailsModel distributionDetails) throws AlertException {
+        //TODO: Review with team, should we call the JiraServerPropertiesFactory.createJiraProperties using a FieldModel?
         JiraServerProperties jiraProperties = jiraServerPropertiesFactory.createJiraProperties();
```

Comment:
The short answer is "no". 

The long answer: 
`jiraServerPropertiesFactory.createJiraProperties()` removes the repetitive work of retrieving information from the database when instantiating a `JiraServerProperties`. By retrieving this ourselves, we would just be creating "WET" code. If the expectation is that a `FieldModel` would be provided through the method signature, that would be muddling the intent of `IssueTrackerProcessorFactory`. An `IssueTrackerProcessorFactory` is part of a pre-defined workflow. The public API of `createProcessor` only demands distribution (job) details because that is the only piece of state that should be "externally controlled". The "internally controlled" state is the global state of Alert (including the global Jira Server config) which is expected to be configured and valid by the time an event reaches a channel.